artifacts.md

```diff
@@ -6,7 +6,7 @@ description: Encapsulating data in Aqueduct
 
 _Artifacts_ are thin wrappers around data objects in Aqueduct. Wrapping data in an Artifact allows Aqueduct to track that artifact across workflow runs help you understand how your data is changing over time.&#x20;
 
-Aqueduct currently only supports `TableArtifact`s, which are encapsulations of tabular or relational data. We plan to add support for other data types in the future.
+Aqueduct currently supports **TableArtifacts**, which are encapsulations of tabular or relational data, and [**ParameterArtifacts**](workflows/parameterizing-a-workflow.md), which wrap mutable json values. We plan to add support for other data types in the future.
```

Comment:
Nit -- I personally prefer having types in teletype font. Is there a reason you'd rather have these bolded?

Reply:
not rly, I just wanted Table and Parameter artifacts to be highlighted the same way and I wasn't sure if I could do that with the latter having a link.

Reply:
Ah, yes, you should be able to [`a link in teletype`]().